mailu/values.yaml

```diff
@@ -61,9 +61,24 @@ mail:
 
 # certmanager settings
 certmanager:
+  enabled: false
+  # leave emty for default value
+  #apiVersion: cert-manager.io/v1alpha3
+  apiVersion:
+  # leave emty for default value. If set one, make sure to set the same for ingress.secretName
+  #secretName: &secretName wildcard-cert-secret
+  secretName:
+  # ClusterIssuer or Issuer
   issuerType: ClusterIssuer
+  # issuer name set in cert manager deployment
   issuerName: letsencrypt
 
+ingress:
```

Comment:
I would definitely add a `annotations` section under ingress to be able to customize it based on the type of ingress.

Reply:
I agree, but please in another issue+PR.